javascript/javascript3/week2/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>JS3-weeK3 Homework</title>
+  </head>
+  <body>
+    <div class="converter">
+      <h1>
+        CURRENCY <br />
+        CALCULATOR
+      </h1>
+
+      <div class="input-data">
+        <div class="amount">
+          <input
+            id="inputEl"
+            type="number"
+            placeholder="Enter amount to convert"
+          />
+        </div>
+
+        <div class="convert-from">
+          <label>Convert From </label>
+          <select id="convertFrom"></select>
+        </div>
+
+        <div class="convert-to">
+          <label>Convert To</label>
+          <select id="convertTo"></select>
+        </div>
+      </div>
+
+      <button id="btn" class="btn">Convert</button>
```

Comment:
You don't need class here, you can use HTML tag for styling
`button {}`
I would say you are also missing styling for the button. :)

Reply:
Fixed.